app/scripts/lib/oauth-client.js

```diff
@@ -39,8 +39,6 @@ function (xhr, OAuthErrors) {
 
     // params = { assertion, client_id, scope }
     getToken: function (params) {
-      /* jshint camelcase: false */
-
```

Comment:
This is interesting... Is ESLint not caring about camelcase things?
Should it? http://eslint.org/docs/rules/camelcase.html